packages/ckeditor5-ui/tests/colortable/colortableview.js

```diff
@@ -343,6 +343,13 @@ describe( 'ColorTableView', () => {
 			expect( colorTableView.colorPickerPageView.items.length ).to.equal( 2 );
 		} );
 
+		it( 'should set selected color after changes in color picker', () => {
+			colorTableView._appendColorPicker();
+			colorTableView.colorPickerPageView.colorPickerView._debounceColorPickerEvent( '#113322' );
```

Comment:
Please don't use private properties in tests unless there's absolutely no other choice. This leads to multitude of tests failing in random places if the internal implementation of a class/component changes in the future. 

You can fire an event that belongs to the public API of the `ColorPickerView`

```
colorPickerView.fire( 'colorSelected', { color: '#113322' } );
```

to test this since all we want to know here is that the data propagation between components exists and works fine

```
this.colorPickerView!.on<ColorPickerColorSelectedEvent>( 'colorSelected', ( evt, data ) => {
	// ... 

	this.set( 'selectedColor', data.color );
} );
```